mappings/empire_v2.0.yaml

```
@@ -0,0 +1,341 @@
+model:
+  - EMPIRE v2.0
+
+native_regions:
+  - Austria
+  - Bosnia and Herzegovina
+  - Belgium
+  - Bulgaria
+  - Switzerland
+  - Czech Republic
+  - Germany
+  - Denmark
+  - Estonia
+  - Spain
+  - Finland
+  - France
+  - United Kingdom
+  - Greece
+  - Croatia
+  - Hungary
+  - Ireland
+  - Italy
+  - Lithuania
+  - Luxembourg
+  - Latvia
+  - North Macedonia
+  - The Netherlands
```

Please follow the convention of nomenclature and the file `European_countries.yaml` in this repository.
```suggestion
  - Netherlands
```